libcodechecker/analyze/log_parser.py

```diff
@@ -326,7 +326,7 @@ def parse_compile_commands_json(logfile, parseLogOptions):
             action.target = compiler_target[results.compiler]
 
         if results.action != option_parser.ActionType.COMPILE:
-            action.skip = True
+            continue
```

Comment:
Do we want to keep the `_skip` attribue of `BuldAction` class? Do we use it in other part of our code?
https://github.com/Ericsson/codechecker/blob/4dcae686dd6d707621d7bd5e160fff56db8f0317/libcodechecker/log/build_action.py#L24